src/main/java/uk/ac/ebi/ampt2d/commons/core/AccessioningRepository.java

```diff
@@ -15,15 +15,13 @@
  * limitations under the License.
  *
  */
-package uk.ac.ebi.ampt2d.accession.common.accessioning;
+package uk.ac.ebi.ampt2d.commons.core;
 
```

Comment:
can it be accessioning/accession.commons.core ?

Reply:
NO.

Reply:
It can't be `accessions.commons` because that will cause conflicts between the accession-commons and amp-t2d-accession packages.  It has been agreed offline to use `commons.accession` as root package, so this one in particular would be `commons.accession.core`.